app/listings/call_logs_listing.rb

```diff
@@ -92,6 +92,14 @@ def listing_account
   column :fail_details, title: "Failure" do |log, value|
     value if log.state == :failed
   end
+  column 'Recordings' do |log|
+    has_recordings = log.recorded_audios.any?
```

Comment:
This generates a N+1 query. It will send a query to the DB for each CallLog shown in the table.

Change the model to use something like

```ruby
CallLog.for_account(listing_account)
  .select("*, (select count(*) from recorded_audios where recorded_audios.call_log_id = call_logs.id) as recorded_audios_count")
  .order('call_logs.id DESC')
```

Then `log.recorded_audios_count` can be used in this block to show the check or not.